core/src/main/java/io/questdb/cutlass/http/processors/JsonQueryProcessorState.java

```diff
@@ -303,10 +306,12 @@ private static void putLong256Value(HttpChunkedResponseSocket socket, Record rec
         socket.put('"');
     }
 
-    private static void putLongValue(HttpChunkedResponseSocket socket, Record rec, int col) {
+    private static void putLongValue(HttpChunkedResponseSocket socket, Record rec, int col, boolean isConsoleRequest) {
         final long l = rec.getLong(col);
         if (l == Long.MIN_VALUE) {
             socket.put("null");
+        } else if (isConsoleRequest) {
```

Comment:
It would be good to keep argument name consistent with arguments and member variable names up the call stack.

Reply:
Yeah, apologize for the oversight